src/lib/components/tablev2/SingleSelectableContent.js

```diff
@@ -150,17 +160,21 @@ export default function SingleSelectableContent({
   );
 
   const scrollbarWidth = useMemo(() => {
-    const scrollDiv = document.createElement('div');
-    scrollDiv.setAttribute(
-      'style',
-      'width: 100px; height: 100px; overflow: scroll; position:absolute; top:-9999px;',
-    );
-    const body = document.body || {};
-    body.appendChild(scrollDiv);
-    const scrollbarWidth = scrollDiv.offsetWidth - scrollDiv.clientWidth;
-    body.removeChild(scrollDiv);
-    return scrollbarWidth;
-  }, []);
+    if (outerRef.current) {
+      const scrollDiv = document.createElement('div');
+      scrollDiv.setAttribute(
+        'style',
+        'width: 100px; height: 100px; overflow: scroll; position:absolute; top:-9999px;',
+      );
+      // $FlowFixMe
+      outerRef.current.appendChild(scrollDiv);
+      const scrollbarWidth = scrollDiv.offsetWidth - scrollDiv.clientWidth;
+      // $FlowFixMe
+      outerRef.current.removeChild(scrollDiv);
+      return scrollbarWidth;
+    }
+    return 0;
+  }, [outerRef.current]);
```

Comment:
Can you check this warning ? Maybe just mentioning outerRef in the dependency array would solve it ?

> React Hook useMemo has an unnecessary dependency: 'outerRef.current'. Either exclude it or remove the dependency array. Mutable values like 'outerRef.current' aren't valid dependencies because mutating them doesn't re-render the component

Reply:
testé et pas approuvé, ça ne marche pas sans le `.current`